ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/MLAG_ODD_ID_L3LEAF1B.cfg

```diff
@@ -99,7 +93,9 @@ mlag configuration
    domain-id MLAG_ODD_ID_L3LEAF1
    local-interface Vlan4094
    peer-address 10.10.255.2
+   peer-address heartbeat 192.168.201.116 vrf MGMT
```

Comment:
the comment states that the peer hearbeat config will not be generated but it is added in the PR - is it expected?

Ah you mean it won't be configured on the peer maybe?
Though this config would then be wrong as the heartbeat will never work. @ClausHolbechArista we probably need to open an issue to track this (as this is existing behavior)

Reply:
We should error if the mgmt_ip is not set and we have heartbeat configured.